kratos/solving_strategies/strategies/explicit_solving_strategy_runge_kutta.h

```diff
@@ -7,12 +7,12 @@
 //  License:		 BSD License
 //					 Kratos default license: kratos/license.txt
 //
-//  Main authors:    Ruben Zorrilla
+//  Main authors:    Ruben Zorrilla, Eduard Gómez
```

Comment:
FYI: https://github.com/KratosMultiphysics/Kratos/wiki/Authorship-of-Kratos-files

Reply:
Didn't know. Changed.